packages/doenetml-worker-rust/lib-doenetml-core/src/core/doenet_ml_core.rs

```diff
@@ -243,6 +244,14 @@ pub struct Action {
     pub args: HashMap<String, Vec<StateVarValue>>,
 }
 
+#[derive(Debug, Deserialize, Serialize, derive_more::TryInto)]
+#[cfg_attr(feature = "web", derive(Tsify))]
+#[cfg_attr(feature = "web", serde(rename_all = "camelCase"))]
+#[cfg_attr(feature = "web", tsify(from_wasm_abi))]
+pub enum AllActions {
```

Comment:
I suggest we move this into `common.rs` of `components`. That way, when creating a new component, you only have to add to `enum`s in one file.

I would also give this a singular name, as an instance of the `enum` actually represents just one action. The problem is that `Action` is already used. Maybe something like `ComponentAction`?

Reply:
Done!